sql/core/src/test/scala/org/apache/spark/sql/execution/vectorized/ColumnVectorSuite.scala

```diff
@@ -130,9 +142,11 @@ class ColumnVectorSuite extends SparkFunSuite with BeforeAndAfterEach {
     }
 
     val array = new ColumnarArray(testVector, 0, 10)
+    val arrayCopy = array.copy()
```

Comment:
can we also test `copy` for non-primitive types?

Reply:
I have added for `String` and `Binary`

Reply:
btw, better to add a new test unit instead of modifying the existing tests?